### PR TITLE
fixes for wp 7.4.1

### PR DESCRIPTION
**Pull request by ZNick1982**
Fixed following parts:

- [x] Inline image insert
- [x] Use only widget jQuery

надо проверить на последней верии WP, и соответственно обновить

- [x] readme.txt
- [x] uploadcare.php
- [x] changelog.md

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -27,19 +29,19 @@ function ucAddImg(fileInfo) {
     if (fileInfo.isImage) {
       var $img = '<img src="' + fileInfo.cdnUrl + '\" alt="' + fileInfo.name + '"/>';
```
Comment (dmitry-mukhin) [inline js/main.js]: `\"`

**Comment by dmitry-mukhin** [comment]
надо проверить на последней верии WP, и соответственно обновить

- [ ] readme.txt
- [ ] uploadcare.php
- [ ] changelog.md

**Comment by ZNick1982** [comment]
Внес изменения в соответсвии с коментариями.

**Comment by khazamov** [comment]
I'm suggesting to review fixes and perform plugin update on Wordpress.com, without waiting for  #18  to be in place.

**Comment by dmitry-mukhin** [comment]
@ZNick1982 see checklist in my prev comment

**Comment by dmitry-mukhin** [comment]
moved checklist to PR description

**Comment by ZNick1982** [comment]
Checklist updated.

**Comment by dmitry-mukhin** [comment]
?

**Comment by ZNick1982** [comment]
didn't found place for update in uploadcare,php
probably this one:
`define('UPLOADCARE_PLUGIN_VERSION', '2.5.1');`

but confused current version: '2.5.1'
The updated version is '2.4.1'

**Comment by dmitry-mukhin** [comment]
theres meta data in the header comments that WP looks at

**Comment by dmitry-mukhin** [comment]
https://developer.wordpress.org/plugins/the-basics/header-requirements/

**Comment by ZNick1982** [comment]
Yes, but also confused the current version, it is does not match with version in changelog.md
Anyway I fixed to the correct one according version on the https://wordpress.org/plugins/uploadcare/

**Comment by dmitry-mukhin** [comment]
https://github.com/uploadcare/uploadcare-wordpress/blob/master/readme.txt#L6

**Comment by ZNick1982** [comment]
Yes, this one missed.
Fixed.

File: readme.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Contributors: grayhound1, dmitry-mukhin
 Tags: media upload, file handling, cdn, storage, facebook, dropbox, instagram, google drive, vk, evernote, box, images, flickr
 Requires at least: 3.5+
-Tested up to: 4.4.2
+Tested up to: 4.7.2
 Stable tag: 2.4.0
```
Comment (dmitry-mukhin) [inline readme.txt]: 2.4.1

File: changelog.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ## Changelog
+**2.4.1**
```
Comment (dmitry-mukhin) [inline changelog.md]: add download link (see below)

File: readme.txt (hunk 2 above)
Comment (ZNick1982) [inline readme.txt]: done

File: changelog.md (hunk 3 above)
Comment (ZNick1982) [inline changelog.md]: done

**Pull request merged**